src/Microsoft.AspNet.Mvc.Core/ModelBinding/CancellationTokenModelBinder.cs

```diff
@@ -17,7 +17,13 @@ public Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContex
             if (bindingContext.ModelType == typeof(CancellationToken))
             {
                 var model = bindingContext.OperationBindingContext.HttpContext.RequestAborted;
-                return Task.FromResult(new ModelBindingResult(model, bindingContext.ModelName, isModelSet: true));
+                var validationNode =
+                    new ModelValidationNode(bindingContext.ModelName, bindingContext.ModelMetadata, model);
```

Comment:
Why is there a validation node here? We don't want to validate a `CancellationToken`